Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -529,6 +529,7 @@ class TabTrayController: UIViewController {
     }
 
     private func openNewTab(request: NSURLRequest? = nil) {
+        view.userInteractionEnabled = false
```

Comment:
Instead of turning off interactions for the whole view, can we disable just the + button? You should be able to reference it from `leftToolbarButtons`.